index.css

```diff
@@ -0,0 +1,58 @@
+* {
```

Comment:
General style comment:

Colours look a bit strange and the text is pretty small

Reply:
Definitely fix this